src/mongo/integration/standalone/dbclient_test.cpp

```diff
@@ -1414,7 +1414,11 @@ namespace {
         if (serverGTE(&c, 2, 4) && (!serverGTE(&c, 2, 7) || kCompiledWithSSL)) {
             createUser(c, TEST_DB, "user3", "password3");
             std::string errmsg;
-            ASSERT_FALSE(c.auth("test", "user3", "notPassword3", errmsg));
+            try {
+                ASSERT_FALSE(c.auth("test", "user3", "notPassword3", errmsg));
+            } catch (const DBException&) {
+                //Expected on 2.2
```

Comment:
Too bad, I feel like very similar fns are already defined elsewhere in our tests